modules/plugins/utility/binds/cheatsheet/config.nix

```diff
@@ -4,15 +4,18 @@
   ...
 }: let
   inherit (lib.modules) mkIf;
-  inherit (lib.nvim.dag) entryAnywhere;
+  inherit (lib.strings) optionalString;
 
   cfg = config.vim.binds.cheatsheet;
 in {
   config = mkIf cfg.enable {
-    vim.startPlugins = ["cheatsheet-nvim"];
+    vim.lazy.plugins.cheatsheet-nvim = {
```

Comment:
It would be good to make the lazy.plugins.* attribute names consistent with the module names (check this for other modules too, I probably won't catch everything)

Reply:
the vim.lazy.plugins.* is significant and must be the pname/name of the plugin package (follows how mnw names the plugin directories)

Reply:
Hmm, that's not very obvious, but I can agree to it